pallets/settlement-groth16/src/lib.rs

```diff
@@ -0,0 +1,114 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
You missed the headers here

Reply:
Added them in https://github.com/HorizenLabs/zkVerify/pull/66/commits/49dfe05d30aee834f76b2af36511bad3e4158d1b